### PR TITLE
[supports-webp] Add libdef for supports-webp@{1,2}.x.x

### PR DESCRIPTION
**Pull request by 102**

### CODE REVIEW

File: definitions/npm/supports-webp_v1.x.x/flow_v0.84.x-/supports-webp_v1.x.x.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+// @flow
+
+declare module "supports-webp" {
+  declare module.exports: boolean;
```
Comment (102) [inline definitions/npm/supports-webp_v1.x.x/flow_v0.84.x-/supports-webp_v1.x.x.js]: https://github.com/fregante/supports-webp/blob/v1.0.6/index.js

File: definitions/npm/supports-webp_v2.x.x/flow_v0.84.x-/supports-webp_v2.x.x.js [hunk 2]
```diff
@@ -0,0 +1,5 @@
+// @flow
+
+declare module "supports-webp" {
+  declare module.exports: Promise<boolean>;
```
Comment (102) [inline definitions/npm/supports-webp_v2.x.x/flow_v0.84.x-/supports-webp_v2.x.x.js]: https://github.com/fregante/supports-webp/blob/v2.0.1/index.js

File: definitions/npm/supports-webp_v1.x.x/flow_v0.84.x-/supports-webp_v1.x.x.js (hunk 1 above)
Comment (pascalduez) [reply definitions/npm/supports-webp_v1.x.x/flow_v0.84.x-/supports-webp_v1.x.x.js]: Please prefer ES modules. Especially because the package is using it.

Comment (102) [reply definitions/npm/supports-webp_v1.x.x/flow_v0.84.x-/supports-webp_v1.x.x.js]:
```suggestion
  declare export default boolean;
```

File: definitions/npm/supports-webp_v2.x.x/flow_v0.84.x-/supports-webp_v2.x.x.js (hunk 2 above)
Comment (102) [reply definitions/npm/supports-webp_v2.x.x/flow_v0.84.x-/supports-webp_v2.x.x.js]:
```suggestion
  declare export default Promise<boolean>;
```

**Pull request merged**